graphics/macgui/macmenu.cpp

```diff
@@ -391,14 +391,11 @@ const Font *MacMenu::getMenuFont() {
 	return _wm->_fontMan->getFont(Graphics::MacFont(kMacFontChicago, 12));
 }
 
-const char *MacMenu::getAcceleratorString(MacMenuSubItem *item, const char *prefix) {
-	static char res[20];
-	*res = 0;
+const Common::String MacMenu::getAcceleratorString(MacMenuSubItem *item, const char *prefix) {
+	if (item->shortcut == 0)
+		return Common::String();
 
-	if (item->shortcut != 0)
-		sprintf(res, "%s%c%c", prefix, (_wm->_fontMan->hasBuiltInFonts() ? '^' : '\x11'), item->shortcut);
-
-	return res;
+	return Common::String::format("%s%c%c", prefix, (_wm->_fontMan->hasBuiltInFonts() ? '^' : '\x11'), item->shortcut);
```

Comment:
Same here. A Common::String variable is still needed here, as Common::String::format() is a static method, so no new instance of a Common::String is created. You could do:

    return Common::String(Common::String::format(...));